provisioning_client/src/prov_transport_amqp_common.c

```diff
@@ -245,6 +277,9 @@ static void on_amqp_send_complete(void* user_ctx, MESSAGE_SEND_RESULT send_resul
         PROV_TRANSPORT_AMQP_INFO* amqp_info = (PROV_TRANSPORT_AMQP_INFO*)user_ctx;
         if (send_result != MESSAGE_SEND_OK)
         {
+            (void)delivery_state;
+            // To be included when a issue in the service is fixed.
```

Comment:
>// To be included when a issue in the service is fixed. [](start = 12, length = 55)

Is this TODO fixed?

Reply:
It's a big part of the feature, but not everything.  They have a bug and they don't want to deploy till after build.

---
In reply to: [281245051](https://github.com/Azure/azure-iot-sdk-c/pull/984#discussion_r281245051) [](ancestors = 281245051)